### PR TITLE
OSDOCS-4717 reminder to leave NetworkManager enabled

### PR DESCRIPTION
**Pull request by jmanthei**
...for the success of the node

Version(s): 4.11, 4.12

Issue: https://issues.redhat.com/browse/OSDOCS-4717

Link to docs preview: https://62380--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/node-tasks.html#rhel-preparing-node_post-install-node-tasks

QE review:
@sdodson 
@cpassare 
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@gpei Can you please provide QE review or point us to the correct contact? Thank you!

**Comment by abhatt-rh** [comment]
> /label merge-review-needed

Hi @jmanthei , I don't see `peer-review` related labels. Wondering if you wanted to request a peer-review instead of merge-review?

File: modules/rhel-preparing-node.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@
 [id="rhel-preparing-node_{context}"]
 = Preparing a RHEL compute node
 
-Before you add a Red Hat Enterprise Linux (RHEL) machine to your {product-title} cluster, you must register each host with Red Hat Subscription Manager (RHSM), attach an active {product-title} subscription, and enable the required repositories.
+Before you add a Red Hat Enterprise Linux (RHEL) machine to your {product-title} cluster, you must register each host with Red Hat Subscription Manager (RHSM), attach an active {product-title} subscription, and enable the required repositories. Ensure `NetworkManager` is enabled and configured to control all interfaces on the host.
```
Comment (tmalove) [inline modules/rhel-preparing-node.adoc:10]:
```suggestion
Before you add a Red Hat Enterprise Linux (RHEL) machine to your {product-title} cluster, you must register each host with Red Hat Subscription Manager (RHSM), attach an active {product-title} subscription, and enable the required repositories. Enable and configure `NetworkManager` to control all interfaces on the host.
```
I suggest using the active voice.

**Review by tmalove — APPROVED** [review]

**Comment by adellape** [comment]
@jmanthei From PR description:

> Version(s): 4.9, 4.10, 4.11, 4.12

I'm guessing this list of versions to cherrypick to only goes to 4.12 because that's what https://issues.redhat.com/browse/OSDOCS-4717 mentioned (which was probably the latest version released when the ticket was opened). But since this PR is making changes to `main`, I would expect to include 4.13 and 4.14, unless there was a specific reason not to (and I haven't found a comment to that effect in the PR/JIRA). Can you confirm that this should get picked all the way to 4.14? Then re-queue for merge. Thank you!

**Comment by jmanthei** [comment]
@adellape I reviewed all of the versions and it can be applied to all of them.

**Comment by kalexand-rh** [comment]
Only versions 4.11+ are still in maintenance. I am removing labels from earlier versions from this PR.

**Comment by bscott-rh** [comment]
Discussed resolving the existing peer review feedback with Jessi via DM

**Pull request merged**